server/api/servarr/sonarr.ts

```diff
@@ -318,39 +326,39 @@ class SonarrAPI extends ServarrBase<{
 
   private buildSeasonList(
     seasons: number[],
-    existingSeasons?: SonarrSeason[]
+    existingSeasons?: SonarrSeason[],
+    autoRequestNewSeasons?: boolean
   ): SonarrSeason[] {
     if (existingSeasons) {
-      const newSeasons = existingSeasons.map((season) => {
+      return existingSeasons.map((season) => {
+        // Monitor requested seasons
         if (seasons.includes(season.seasonNumber)) {
           season.monitored = true;
+        } else {
+          // Set future seasons' monitoring based on autoRequestNewSeasons
+          season.monitored = autoRequestNewSeasons !== false;
         }
         return season;
       });
-
-      return newSeasons;
     }
 
-    const newSeasons = seasons.map(
-      (seasonNumber): SonarrSeason => ({
-        seasonNumber,
-        monitored: true,
-      })
-    );
-
-    return newSeasons;
+    // If no existing seasons, monitor only the requested seasons
+    return seasons.map((seasonNumber) => ({
+      seasonNumber,
+      monitored: true,
+    }));
   }
 
-  public removeSerie = async (serieId: number): Promise<void> => {
+  public removeSeries = async (serieId: number): Promise<void> => {
     try {
       const { id, title } = await this.getSeriesByTvdbId(serieId);
       await this.delete(`/series/${id}`, {
         deleteFiles: 'true',
         addImportExclusion: 'false',
       });
-      logger.info(`[Radarr] Removed serie ${title}`);
+      logger.info(`[Sonarr] Removed series ${title}`);
```

Comment:
No "s" here?

Reply:
Wdym?

Reply:
Oh, that's my bad, i mixed up English and French